lib/src/xcvbnm_impl.dart

```diff
@@ -0,0 +1,24 @@
+library xcvbnm.impl;
```

Comment:
What is the idea behind separating `impl` and `common`? I'm not quite sure I understand it